docs/api/api.md

```diff
@@ -359,6 +384,57 @@ From                                                                          To
 | |`"from": 1536106931, "to": "12aH", "timeZone": "GMT+2"`|(this timestamp corresponds to 05-09-2018, 12:22:11 GMT+2) This sets the ending date to 05-09-2018, 12:00:00 GMT+2 (05-09-2018, 10:00:00 UTC)
 | |`"from": "5ah", "to": "21ah"`|This sets the starting date to 07-10-2018, 19:00:00 UTC and the ending date to 07-10-2018, 21:00:00 UTC
 
+### Retries
+
+The Client supports a retry mechanism for **connectivity issues only**. The retry mechanism is disabled by default (the `retries` parameter is set to `0`by default in the `Client` constructor).
+
+In order to enable it, you must set up the `retries` parameter with a value bigger or equals than `1`. The regular action/command is not considered a retry, only the additional attempts.
+
+
+### Keep Alive mechanism support
+
+Some queries may require a big time to start returning data, because of the calculations required, the load of the platform or just because the data belongs to the future (until this future moment is not reached the platform cannot return values).
```

Comment:
Not sure if I would write this: `because the data belongs to the future (until this future moment is not reached the platform cannot return values).`

Maybe more simple `because the data hasn't been ingested yet.`